Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mcr.microsoft.com/dotnet/core/runtime:3.1

```

Comment:
I don't think `ubuntu:latest` has dotnet tools built in, so using this image *should* work better

Reply:
Yep, this is good anyway